atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -189,12 +189,17 @@ def set_operators_metadata(self, extra, worker_metadatas):
                 }
                 for p in op_related_images['pullspecs']
             ]
-            extra['image']['operator_manifests'] = {
+            koji_operator_manifests = {
                 'related_images': {
                     'pullspecs': pullspecs,
                     'created_by_osbs': op_related_images['created_by_osbs'],
                 }
             }
+            custom_csv_mods = op_bundle_metadata.get('custom_csv_modifications_applied')
```

Comment:
I'd default to False, and always set 'custom_csv_modifications_applied'

Reply:
reworked